include/utility/dbus_utility.hpp

```diff
@@ -172,6 +172,19 @@ inline bool callPIM(types::ObjectMap&& objectMap)
 {
     try
     {
+        for (const auto& l_objectKeyValue : objectMap)
+        {
+            auto l_nodeHandle = objectMap.extract(l_objectKeyValue.first);
+            std::string l_objectPath = l_nodeHandle.key().str;
```

Comment:
Don;t see a use of this variable. Can we get rid of this copy here.

Reply:
Done